packages/deployer/test/helpers.js

```diff
@@ -67,22 +60,6 @@ function _getEnvironmentPath(fixtureProjectName) {
   return pathname;
 }
 
-function _getEnvironmentInitializer(fixtureProjectName) {
-  const initializerPath = `${_getEnvironmentPath(fixtureProjectName)}/test/helpers/initializer`;
-
-  try {
-    return require(initializerPath);
-  } catch (err) {
-    if (err.code === 'MODULE_NOT_FOUND') {
-      throw new Error(
-        `Sample project didn't define any tests environment initializer helper: ${initializerPath}`
-      );
-    }
-
-    throw err;
-  }
-}
-
```

Comment:
I'm removing all the initialization logic from here, as we are only needing it once on a single test, so, there's no need to try to load it always for now.

Reply:
It's only need once up to now, but this is going to be used on almost all integration tests for the end user projects since they most likely will require an initialized system (i.e. owned)